docs/guides/bridges/slack-to-confluent.md

```diff
@@ -17,7 +17,7 @@ You will need:
   - [Subscribe to Slack Events](#subscribe-to-slack-events)
   - [Testing the Integration](#testing-the-integration)
 
-## Create Confluent user
+## Create Confluent User
```

Comment:
Due to the way the ToC is generated, you will need to modify L14 to ensure the case matches as well.

Reply:
I changed this :)